learners/setup.md

```diff
@@ -59,6 +91,8 @@ OpenRefine is safe to run.
    selecting "Extract…". Name that directory something like OpenRefine.
    
    :::::::::::::::::::::::::::::::::::::::::  callout
+   #### Long paths
```

Comment:
```suggestion
   
   ## Long paths
```

I don't think this extra newline is _necessary_ but I do think it makes the source files more readable and it makes this box consistent with the rest of your lesson. We really need to start a style guide for writing lesson source Markdown...